src/Components/AddItem.jsx

```diff
@@ -1,5 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import firebase from 'firebase';
+import { db } from '../lib/firebase';
 
 export default function AddItem() {
-  return <div>Hi from Add an Item View!</div>;
+  const [token, setToken] = useState('test-token');
+  const { register, handleSubmit, watch, errors } = useForm();
```

Comment:
This isn't a blocker, but we noticed `watch` wasn't being used in the code. Is this something we can remove?